website/content/docs/app-config/index.mdx

```diff
@@ -57,7 +57,7 @@ $ waypoint config set -app web PORT=8080
 
 Configuration can also be set directly in the `waypoint.hcl` file using
 the [`config` stanza](/docs/waypoint-hcl/config). The example below achieves
-the same thing as the `waypoint config set` example in the section above.
+the same thing as the [`waypoint config set`](/commands/config-set) example in the section above.
```

Comment:
That's a great idea

Reply:
I decided to link just `section above` instead of the entire phrase as suggested since there isn't a way to link directly to an example. I can change it to your suggestion if you would prefer.

Addressed in [ba6b389](https://github.com/hashicorp/waypoint/pull/3876/commits/ba6b3892f34f07a8a865612f24998183665edfa8)

Reply:
Looks great 👍